dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/package-info.java

```diff
@@ -12,7 +12,7 @@
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 /**
- * @version 1.10.0
+ * @version 1.11.0
  */
-@org.osgi.annotation.versioning.Version("1.10.0")
+@org.osgi.annotation.versioning.Version("1.11.0")
```

Comment:
I believe if you run a build it is going to tell you that you also need to update the version of the javadoc jar.